src/domain/collection/curatedEventList/CuratedEventList.tsx

```diff
@@ -33,10 +38,25 @@ const CuratedEventList: React.FC<Props> = ({ collection }) => {
     [collection.curatedEvents]
   );
 
-  const { data: eventsData, loading } = useEventsByIdsQuery({
-    variables: { ids: eventIds, include: ['keywords', 'location'] },
+  const queryVariables = {
+    ids: eventIds.slice(0, PAGE_SIZE),
+    include: ['keywords', 'location'],
+  };
+
+  const { data: eventsData, loading, fetchMore } = useEventsByIdsQuery({
+    variables: queryVariables,
   });
 
+  const page = React.useRef(
```

Comment:
Small thing but would this be a bit more readable like this?
```
    // if eventsByIds is available on first render, they are coming from cache
const eventsCount = eventsData?.eventsByIds.length ?? 0;

    // Initialize page number based on its length
const pageNumber = React.useRef(eventsCount > 0 ? Math.ceil(eventsCount / PAGE_SIZE : 1);
```
`pageNumber` is more readable for me than `page` also... `page` reminds me of a page-object first instead of number. Also `INITIAL_PAGE` could be `INITIAL_PAGE_NUMBER` or just `1` because it is obvious first page number.

Reply:
yes pageNumber is better and I agree tgat PAGE_NUMBER constant might be redundant